locales/en/notifications.json

```diff
@@ -73,7 +73,8 @@
   "ParsingCsv": "Parsing CSV file",
   "ParsingCsvWithLineNumber": "Parsing CSV file row",
   "StartingRoutingParallelServers": "Starting parallel routing servers",
-  "StoppingRoutingParallelServers": "Stoping parallel routing servers",
+  "StoppingRoutingParallelServers": "Stopping parallel routing servers",
```

Comment:
Why reversing the name between the attribute and the text?

Reply:
What do you mean? I just fixed a typo in stoping here